examples/unconditional_image_generation/train_unconditional.py

```diff
@@ -156,7 +156,6 @@ def parse_args():
     parser.add_argument(
         "--use_ema",
         action="store_true",
-        default=True,
         help="Whether to use Exponential Moving Average for the final model weights.",
     )
```

Comment:
Also removing the wrong default for consistency with other scripts (issue #1654)

Reply:
small breaking change but think that's fine!